test/unit/utils.html

```diff
@@ -73,30 +73,116 @@
       setTimeout(function() {
         assert.equal(called, 1, 'debounce should be called exactly once');
         done();
-      }, 50);
+      });
 
     });
 
     test('debounce (wait)', function(done) {
 
       var called = 0;
-      var now = Date.now();
       var cb = function() {
         called++;
       };
 
       window.el1.debounce('foo', cb);
-      window.el1.debounce('foo', cb, 100);
+      window.el1.debounce('foo', cb, 50);
       window.el1.debounce('foo', cb, 100);
 
       setTimeout(function() {
-        assert.equal(called, 1, 'debounce should be called exactly once');
-        assert(Date.now() - now > 100, 'debounce should be called after at least 100ms');
+        assert.equal(called, 0, 'callback is not called yet');
+      }, 50);
```

Comment:
The lines above are replacing the the assertion that was deleted in the first commit of this PR.